internal/manifests/collector/container_test.go

```diff
@@ -181,14 +181,19 @@ service:
 			description: "prometheus exporter",
 			specConfig: `exporters:
     prometheus:
-        endpoint: "0.0.0.0:9090"`,
+        endpoint: "0.0.0.0:9090"
+service:
+    pipelines:
+        metrics:
+            exporters: [prometheus]
+`,
+
 			specPorts: []corev1.ServicePort{},
 			expectedPorts: []corev1.ContainerPort{
 				metricContainerPort,
 				{
 					Name:          "prometheus",
 					ContainerPort: 9090,
-					Protocol:      corev1.ProtocolTCP,
```

Comment:
do we not need the protocol to be set anymore?

Reply:
It's TCP by default.